airbyte-webapp/src/packages/cloud/components/experiments/FreeConnectorProgram/InlineEnrollmentCallout.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { PropsWithChildren } from "react";
+import { FormattedMessage } from "react-intl";
+
+import { Callout } from "components/ui/Callout";
+import { Text } from "components/ui/Text";
+
+import { useShowEnrollmentModal } from "./EnrollmentModal";
+import styles from "./InlineEnrollmentCallout.module.scss";
+
+export const EnrollLink: React.FC<PropsWithChildren<unknown>> = ({ children }) => {
+  const { showEnrollmentModal } = useShowEnrollmentModal();
+
+  const onEnrollClick = () => {
+    showEnrollmentModal();
+  };
+
+  return (
+    <button onClick={onEnrollClick} className={styles.enrollLink}>
+      {children}
+    </button>
+  );
+};
+export const InlineEnrollmentCallout: React.FC = () => {
+  return (
+    <Callout variant="info" className={styles.container}>
+      <Text size="sm">
+        <FormattedMessage
+          id="freeConnectorProgram.youCanEnroll"
+          values={{
+            enrollLink: (content: React.ReactNode) => <EnrollLink>{content}</EnrollLink>,
+            freeText: (content: React.ReactNode) => (
+              <Text as="span" size="sm" bold className={styles.freeText}>
+                {content}
+              </Text>
+            ),
+          }}
+        />
```

Comment:
Should `Free Connector Program` also be bold? Looks like it should according to this [figma link](https://www.figma.com/file/wihAJg5mkp61I7lLHs0IpV/FREE-CONNECTOR-PROGRAM?node-id=1080%3A16332&t=8sRgqgD8AIppl0gJ-0), but I'm a little lost on what the latest designs are

<img width="850" alt="image" src="https://user-images.githubusercontent.com/7550957/213144464-f484a6cb-790d-49ca-b333-aef52a0f55bb.png">